Objects/sliceobject.c

```diff
@@ -590,13 +590,17 @@ slice_richcompare(PyObject *v, PyObject *w, int op)
     }
 
     t1 = PyTuple_New(3);
-    if (t1 == NULL)
+    if (t1 == NULL) {
         return NULL;
+    }
+    PyObject_GC_UnTrack(t1);
```

Comment:
Does it makes sense to add a small comment here explaining why we untrack the tuples?